pkg/packagekit/updates.jsx

```diff
@@ -489,6 +489,9 @@ class ApplyUpdates extends React.Component {
                 // small timeout to avoid excessive overlaps from the next PackageKit progress signal
                 PK.call(transactionPath, "org.freedesktop.DBus.Properties", "GetAll", [PK.transactionInterface], { timeout: 500 })
                         .done(reply => {
+                            if (this._mounted === false)
```

Comment:
`this._mounted` is never set to true -- you want to do that at the start of componentDidMount(). `_mounted` seems to be a common convention (judging by the many stackoverflow and other articles), but it's not a React builtin variable.

Also, please replace `mounted === false` with `!mounted`, it's a simple boolean.

Reply:
This works however because of the === which compares also types apart from value. do undefined !== false. Since you find it confusing I will adjust.

Reply:
Right, I know that distinction -- but AFAICS we never want to explicitly set it to `undefined` or `null`, we are only interested in in true/false here.